src/universal/modules/meeting/containers/MeetingContainer/MeetingContainer.js

```diff
@@ -83,6 +85,18 @@ export default class MeetingContainer extends Component {
     }
   }
 
+  componentWillUnmount() {
+    // Time for clean up!
+    this.resetContainer();
+  }
+
+  resetContainer() {
+    const {dispatch, teamSub, memberSub} = this.props;
+    teamSub.unsubscribe();
```

Comment:
Ah. That makes sense. I wasn't quite sure how that worked – new interface, new things to learn :)

Let's say we wanted to be kind to the server and shut down the changefeed – would an unsubscribe make sense then?